### PR TITLE
Add tests per endpoint

### PR DESCRIPTION
**Pull request by ivanortegaalba**

### CODE REVIEW

**Comment by martacolombas** [comment]
Nice to see we have tests already! 🙌

File: index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config();
 }
 const express = require('express');
-const app = express();
+const app = new express();
```
Comment (martacolombas) [inline index.js:5]: New is not required as per the documentation, why are we adding it here? https://expressjs.com/en/starter/hello-world.html

File: index.spec.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+describe('/users', () => {
+  const endpoint = '/users';
+  let user = {};
+  let hospital = {};
+
+  beforeAll(async () => {
+    hospital = await Hospital.create({
+      name: 'Plural Engine Hospital',
+      address: 'C/ Pau Alsina 123',
```
Comment (martacolombas) [inline index.spec.js:30]: I’d rather very much we didn’t push real information. We need to delete this commit history

File: index.js (hunk 1 above)
Comment (ivanortegaalba) [reply index.js:5]: You're right! I found it checking other examples, and I forgot to remove it. Good catch!
```suggestion
const app = express();
```

File: index.spec.js (hunk 2 above)
Comment (ivanortegaalba) [reply index.spec.js:30]:
```suggestion
      address: 'Lolipop street',
```

**Pull request merged**